sdk/contentsafety/ai-content-safety-rest/samples-dev/sampleManageBlocklist.ts

```diff
@@ -0,0 +1,236 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Demonstrates how to manage blocklist.
+ */
+
+import ContentSafetyClient, {
+  CreateOrUpdateTextBlocklistParameters,
+  AddBlockItemsParameters,
+  AnalyzeTextParameters,
+  RemoveBlockItemsParameters,
+  isUnexpected
+} from "@azure-rest/ai-content-safety";
+import { AzureKeyCredential } from "@azure/core-auth";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const endpoint = process.env["CONTENT_SAFETY_ENDPOINT"] || "<endpoint>";
+const key = process.env["CONTENT_SAFETY_API_KEY"] || "<key>";
+
+const credential = new AzureKeyCredential(key);
+const client = ContentSafetyClient(endpoint, credential);
+
+async function createOrUpdateTextBlocklist() {
+  const blocklistName = "TestBlocklist";
+  const blocklistDescription = "Test blocklist management.";
+  const createOrUpdateTextBlocklistParameters: CreateOrUpdateTextBlocklistParameters = {
+    contentType: "application/merge-patch+json",
+    body: {
+      description: blocklistDescription,
+    }
+  }
+
+  const result = await client.path("/text/blocklists/{blocklistName}", blocklistName).patch(createOrUpdateTextBlocklistParameters);
+
+  if (isUnexpected(result)) {
+    throw result;
+  }
+
+  console.log("Blocklist created or updated: Name", result.body.blocklistName, ", Description: ", result.body.description);
+}
+
+async function addBlockItems() {
+  const blocklistName = "TestBlocklist";
+  const blockItemText1 = "k*ll";
+  const blockItemText2 = "h*te";
+  const addBlockItemsParameters: AddBlockItemsParameters = {
+    body: {
+      blockItems: [
+        {
+          description: "Test block item 1",
+          text: blockItemText1
+        },
+        {
+          description: "Test block item 2",
+          text: blockItemText2
+        }
+      ]
+    }
+  };
+
+  const result = await client.path("/text/blocklists/{blocklistName}:addBlockItems", blocklistName).post(addBlockItemsParameters);
+
+  if (isUnexpected(result)) {
+    throw result;
+  }
+
+  console.log("Block items added: ");
+  if (result.body.value) {
+    for (const blockItem of result.body.value) {
+      console.log("BlockItemId: ", blockItem.blockItemId, ", Text: ", blockItem.text, ", Description: ", blockItem.description);
+    }
+  }
+}
+
+async function analyzeTextWithBlocklists() {
+  const blocklistName = "TestBlocklist";
+  const inputText = "I h*te you and I want to k*ll you.";
```

Comment:
Just checking if this is okay ?

Reply:
We need to test blocklist here, and use * for masking. Is that acceptable?

Reply:
Updated these cases